t/app/controller/waste_kands_bulky.t

```diff
@@ -474,16 +479,17 @@ FixMyStreet::override_config {
 
     subtest 'Bulky goods email confirmation and reminders' => sub {
         my $report = FixMyStreet::DB->resultset("Problem")->search(undef, { order_by => { -desc => 'id' } })->first;
-        $report->confirmed('2023-08-30T00:00:00');
-        $report->update;
+        my $today = $report->confirmed->strftime('%d %B');
         my $id = $report->id;
         subtest 'Email confirmation of booking' => sub {
+            $mech->clear_emails_ok;
             FixMyStreet::Script::Reports::send();
             my @emails = $mech->get_email;
-            is $emails[1]->header('Subject'), "Bulky waste collection service - reference RBK-$id";
-            my $confirmation_email_txt = $mech->get_text_body_from_email($emails[1]);
-            my $confirmation_email_html = $mech->get_html_body_from_email($emails[1]);
-            like $confirmation_email_txt, qr/Date booking made: 30 August/, 'Includes booking date';
+            is scalar @emails, 0, "No confirmation email sent as mail sent at payment success for Kingston";
```

Comment:
I think this check is in the wrong place, I'm afraid, we need to check that when it sends before payment that an email isn't sent, we know it won't be sent now as the previous test where catch_email was set sent it then.